src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly defined in the `./components/Person/Person` file and that it can handle the props being passed to it. Verify that the component is exported correctly from that file.